libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m

```diff
@@ -318,12 +335,14 @@ - (void)validatePasscodeConfirmed:(NSString *)validPasscode
 {
     [self resetReaminingAttemps];
     [self.verifyDelegate passcodeVerified];
+    UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, [SFSDKResourceUtils localizedString:@"accessibilityUnlockAnnouncement"]);
```

Comment:
Should we do this only if UIAccessibilityIsVoiceOverRunning()

Reply:
We could.  But it should do nothing if voice over is off, right?